### DISCUSSION TITLE
Issue when installing Python additional dependencies

### DISCUSSION THREAD

**Discussion by DboubaMefteh** [zephyrproject-rtos/zephyr#45308] [General]
Hi,
I am following this [getting started guidelines](https://docs.zephyrproject.org/latest/develop/getting_started/index.html), when I try to install Python additional dependencies, I am getting an error shows this below output: 
**_Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-_w135gbm/libusb-package/_**
How could I bpass this issue,
Best regards
Mefteh

**Comment by henrikbrixandersen** [comment]
Please include information about your environment (host OS) and full error output.
